source/nvwave.py

```diff
@@ -530,6 +561,18 @@ def _idleCheck(cls):
 fileWavePlayerThread: threading.Thread | None = None
 
 
+def _onPreSpeak(speechSequence: SpeechSequence):
```

Comment:
can this be made function of `WasapiWavePlayer` for similar reasoning as multiple instances. Alternatively, pass in `_isLeadingSilenceInserted` from a wrapper function in `WasapiWavePlayer`